From 2c2cd7a2bc710e60211e0956dda0d3234e794fac Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Sun, 4 Apr 2021 14:33:24 +0300
Subject: [PATCH 1410/1921] arm64: defconfig: marvell: enable PCIe Endpoint
 support

Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Change-Id: I0222a1c15d0f338aea6a341686d768c5f746b613
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/49162
Tested-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-by: Ofer Heifetz <oferh@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/configs/marvell_v8_sdk_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/marvell_v8_sdk_defconfig b/arch/arm64/configs/marvell_v8_sdk_defconfig
index 365f227a5e9f..ebeaf3a6b7da 100644
--- a/arch/arm64/configs/marvell_v8_sdk_defconfig
+++ b/arch/arm64/configs/marvell_v8_sdk_defconfig
@@ -199,6 +199,8 @@ CONFIG_PCIE_ARMADA_8K=y
 CONFIG_PCIE_KIRIN=y
 CONFIG_PCIE_HISI_STB=y
 CONFIG_PCIE_TEGRA194=m
+CONFIG_PCI_ENDPOINT=y
+CONFIG_PCIE_ARMADA_DW_EP=y
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
 CONFIG_FW_LOADER_USER_HELPER=y
-- 
2.31.1

